failures when running the tests

When trying to run the tests on nixos (with archives fetched from pypy), I have these failures

```
==================================== ERRORS ====================================
____________________ ERROR collecting test/test_document.py ____________________
ImportError while importing test module '/tmp/nix-build-python3.6-python-language-server-0.15.0.drv-0/python-language-server-0.15.0/test/test_document.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
test/test_document.py:3: in <module>
    from test.fixtures import DOC_URI, DOC
E   ModuleNotFoundError: No module named 'test.fixtures'
__________________ ERROR collecting test/test_rpc_manager.py ___________________
ImportError while importing test module '/tmp/nix-build-python3.6-python-language-server-0.15.0.drv-0/python-language-server-0.15.0/test/test_rpc_manager.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
test/test_rpc_manager.py:3: in <module>
    from test.fixtures import BASE_HANDLED_RESPONSE
E   ModuleNotFoundError: No module named 'test.fixtures'
______________________ ERROR collecting test/test_uris.py ______________________
ImportError while importing test module '/tmp/nix-build-python3.6-python-language-server-0.15.0.drv-0/python-language-server-0.15.0/test/test_uris.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
test/test_uris.py:2: in <module>
    from test import unix_only, windows_only
E   ImportError: cannot import name 'unix_only'
!!!!!!!!!!!!!!!!!!! Interrupted: 3 errors during collection !!!!!!!!!!!!!!!!!!!!
=========================== 3 error in 0.65 seconds ============================
```

if I list the content of the test folder: 

```
drwxr-xr-x 3 nixbld1 nixbld 4096 02-22 09:16 plugins
drwxr-xr-x 2 nixbld1 nixbld 4096 02-22 09:16 __pycache__
-rw-r--r-- 1 nixbld1 nixbld 2938 02-20 22:04 test_document.py
-rw-r--r-- 1 nixbld1 nixbld 2653 02-20 22:04 test_json_rpc_server.py
-rw-r--r-- 1 nixbld1 nixbld 1760 02-20 22:04 test_language_server.py
-rw-r--r-- 1 nixbld1 nixbld 7125 02-20 22:04 test_rpc_manager.py
-rw-r--r-- 1 nixbld1 nixbld 1455 02-20 22:04 test_uris.py
-rw-r--r-- 1 nixbld1 nixbld 1434 02-20 22:04 test_utils.py
-rw-r--r-- 1 nixbld1 nixbld 1448 02-20 22:04 test_workspace.py
```
it misses the fixtures.py file that appears on github for instance

--- a/MANIFEST.in
+++ b/MANIFEST.in
@@ -2,3 +2,4 @@ include README.rst
 include versioneer.py
 include pyls/_version.py
 include LICENSE
+recursive-include test *.py

